Create new courses with minimal content (#11124)
PrairieLearn/PrairieLearn: templateCourse/README.md

Reviewer: cc @mfsilva22 @mwest1066 seeking input here - is there anything else you'd like to change, as long as we're taking some time to refine this file?

Reviewer 2: Lots of these links look like they might bit-rot over time. My suggestions for the three sections:
* Live version link --> this seems good and stable, leave it as is.
* Getting started --> this is presumably going to be obsoleted by the new onboarding checklist and other improvements. I think we should delete this whole section.
* Getting help --> Maybe replace this entire section with a link to https://www.prairielearn.com/support, which is a stable link whose contents we will always keep up to date.

Reviewer: These changes sound good to me.

Author: These changes sound good to me as well, @mwest1066 . Here is the updated `README.md`: 

https://github.com/PrairieLearn/PrairieLearn/blob/708da3b3c53faf893372d4593ad11fdf45a62eff/templateCourse/README.md?plain=1#L1-L10

Reviewer 2: Thanks!

> ## PrairieLearn Course Repository
> 
> ### Welcome to PrairieLearn!
> 
> The content for your course is stored within this repository.
> You can see and edit a live version at [https://prairielearn.com](https://prairielearn.com)
> 
> ### Getting Help
> 
> Need help understanding an error or creating content? Visit [https://www.prairielearn.com/support](https://www.prairielearn.com/support)

Reviewer: A few tweaks, the main one is not immediately mentioning errors here. People _might_ hit them, but I think having that be one of the first/only things in the readme could give the impression that errors are going to be very common?

```suggestion
# Welcome to PrairieLearn!

The content for your course is stored within this repository. You can see and edit a live version at [https://prairielearn.com](https://prairielearn.com).

## Support

For help setting up your course or creating content, visit [https://www.prairielearn.com/support](https://www.prairielearn.com/support).
```

@mwest1066 do these changes seem reasonable? We're probably into bikeshedding territory here.

Author: This version looks better to me, @nwalters512 . I'll commit the suggestion if you think it's ready, @mwest1066 .

Reviewer 2: lgtm!

## PrairieLearn Course Repository

### Welcome to PrairieLearn!

The content for your course is stored within this repository.
You can see and edit a live version at [https://prairielearn.com](https://prairielearn.com)

### Getting Started

> Learn how to create your first questions and assessments using our [Get Started](https://prairielearn.readthedocs.io/en/latest/getStarted/) tutorial.

Reviewer: Nit, we now call this the "quick start" tutorial in the docs.

```suggestion
Learn how to create your first questions and assessments using the [quick start tutorial](https://prairielearn.readthedocs.io/en/latest/getStarted/).
```

### Getting Help

Need help understanding an error or creating content? Check out one of the following resources:

> 1. Real-time help assistance on PrairieLearn's [Slack `#pl-help` channel](https://prairielearn.com/slack).

Reviewer: ```suggestion
1. Get real-time assistance in the [`#pl-help` Slack channel](https://prairielearn.com/slack).
```

> 1. Looking up the question on our Frequently Asked Question (FAQ) pages: [readthedocs](https://prairielearn.readthedocs.io/en/latest/faq/) and [github-discussions](https://github.com/PrairieLearn/PrairieLearn/discussions/categories/q-a)

Reviewer: ```suggestion
1. Check out the Frequently Asked Question (FAQ) pages in the [documentation](https://prairielearn.readthedocs.io/en/latest/faq/) or on [GitHub discussions](https://github.com/PrairieLearn/PrairieLearn/discussions/categories/q-a).
```

> 1. Attend virtual office hours, which are announced at the start of the
>    semester on PrairieLearn's [Slack](https://prairielearn.com/slack)

Reviewer: ```suggestion
1. Attend virtual office hours, which are announced at the start of the
   semester on PrairieLearn's [Slack](https://prairielearn.com/slack).
```
